plugin/src/main/java/net/dzikoysk/funnyguilds/config/PluginConfiguration.java

```diff
@@ -1173,4 +1177,44 @@ public void loadProcessedProperties() {
         MYSQL
     }
 
+    public static int getIndex(String text) {
```

Comment:
Po co to wrzucać jeszcze do tego configu, za krótki był wcześniej już :/ Raczej starałbym sie wywalać rzeczy stąd niż przenosić. + nazwa getIndex bez contextu RankUtils nic nie mówi o przeznaczeniu tej metody

Reply:
Nie wiem gdzie przenieść tą metode :/